### PR TITLE
Psay improve

### PR DESCRIPTION
**Pull request by Rushaway**

### CODE REVIEW

**Comment by maxime1907** [comment]
why do we need this ?

it says private for a reason, plus we can already see it in logs if really needed

File: addons/sourcemod/scripting/CustomChatColors.sp [hunk 1]
```diff
@@ -223,6 +224,7 @@ public void OnPluginStart()
 	g_cSmNameColor = CreateConVar("sm_ccc_sm_name_color", "{fullred}", "Color used for SM player name", FCVAR_REPLICATED);
 	g_cSmChatColor = CreateConVar("sm_ccc_sm_chat_color", "{cyan}", "Color used for SM chat", FCVAR_REPLICATED);
 	g_cvPsayCooldown = CreateConVar("sm_ccc_psay_cooldown", "4", "Cooldown between two usage of sm_psay", FCVAR_REPLICATED);
+	g_cvPsayPrivacy = CreateConVar("sm_ccc_psay_privacy", "0", "Hide to admins all usage of sm_psay", FCVAR_REPLICATED);
```
Comment (maxime1907) [inline addons/sourcemod/scripting/CustomChatColors.sp:227]: 1 by default

**Comment by Rushaway** [comment]
> why do we need this ?
> 
> it says private for a reason, plus we can already see it in logs if really needed

It's too hard for admins to check logs in the same time of playing for find players who use psay for running others players experience. (spam, prepare trim etc..)

**Comment by maxime1907** [comment]
ok for this behavior, we are gonna add a message in ServerAdvertisement to let the players know that they should never share confidential information such as passwords, IP adresses, Real adresses etc

**Pull request merged**